CHANGELOG.md

```diff
@@ -3,6 +3,88 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.11.1
+
+_Jul 25, 2024_
+
+We'd like to offer a big thanks to the 18 contributors who made this release possible. Here are some highlights ✨:
+
+- 🌍 Add Icelandic (is-IS) and Norwegian Nynorsk (nn-NO) locales on the Data Grid
```

Comment:
Let me know if there's something else that should be highlighted.

Reply:
Maybe "Allow `Zoom` to be controllable"? 🤔 
cc @mui/xcharts